
CommandNotFoundError: Your shell has not been properly configured to use 'conda activate'.
To initialize your shell, run

    $ conda init <SHELL_NAME>

Currently supported shells are:
  - bash
  - fish
  - tcsh
  - xonsh
  - zsh
  - powershell

See 'conda init --help' for more information and options.

IMPORTANT: You may need to close and restart your shell after running 'conda init'.


cat: write error: Broken pipe
2023-01-26 21:12:54.604571
2023-01-26 21:12:54.927013
2023-01-26 21:12:55.153612

real	0m1.466s
user	0m1.276s
sys	0m0.165s
2023-01-26 21:13:06.365084
2023-01-26 21:13:06.568480
2023-01-26 21:13:06.719329

real	0m1.260s
user	0m1.084s
sys	0m0.153s
cat: write error: Broken pipe
2023-01-26 21:13:17.928078
2023-01-26 21:13:18.112872
2023-01-26 21:13:18.236022

real	0m1.216s
user	0m1.033s
sys	0m0.162s
cat: write error: Broken pipe
2023-01-26 21:13:29.491251
2023-01-26 21:13:29.689776
2023-01-26 21:13:29.816257

real	0m1.285s
user	0m1.120s
sys	0m0.134s
2023-01-26 21:13:41.057484
2023-01-26 21:13:41.347584
2023-01-26 21:13:41.513922

real	0m1.404s
user	0m1.235s
sys	0m0.147s
cat: write error: Broken pipe
2023-01-26 21:13:52.802093
2023-01-26 21:13:53.344162
2023-01-26 21:13:53.779840

real	0m1.973s
user	0m1.782s
sys	0m0.167s
2023-01-26 21:14:05.264944
2023-01-26 21:14:07.222801
2023-01-26 21:14:09.447923

real	0m5.373s
user	0m4.980s
sys	0m0.354s
2023-01-26 21:14:20.665483
2023-01-26 21:14:20.821565
2023-01-26 21:14:20.924177

real	0m1.171s
user	0m1.016s
sys	0m0.131s
mkdir: cannot create directory ‘SCODE_D_2’: File exists
┌ Warning: `CSV.read(input; kw...)` is deprecated in favor of `using DataFrames; CSV.read(input, DataFrame; kw...)
│   caller = ip:0x0
└ @ Core :-1
┌ Warning: Assignment to `WZ` in soft scope is ambiguous because a global variable by the same name exists: `WZ` will be treated as a new local. Disambiguate by using `local WZ` to suppress this warning or `global WZ` to assign to the existing global variable.
└ @ ~/stephenz/locaTE-paper/tools/SCODE/SCODE.jl:69
┌ Warning: `writeheader=false` is deprecated in favor of `header=false`
│   caller = ip:0x0
└ @ Core :-1
a
mkdir: cannot create directory ‘SCODE_D_4’: File exists
┌ Warning: `CSV.read(input; kw...)` is deprecated in favor of `using DataFrames; CSV.read(input, DataFrame; kw...)
│   caller = ip:0x0
└ @ Core :-1
┌ Warning: Assignment to `WZ` in soft scope is ambiguous because a global variable by the same name exists: `WZ` will be treated as a new local. Disambiguate by using `local WZ` to suppress this warning or `global WZ` to assign to the existing global variable.
└ @ ~/stephenz/locaTE-paper/tools/SCODE/SCODE.jl:69
┌ Warning: `writeheader=false` is deprecated in favor of `header=false`
│   caller = ip:0x0
└ @ Core :-1
a
mkdir: cannot create directory ‘SCODE_D_6’: File exists
┌ Warning: `CSV.read(input; kw...)` is deprecated in favor of `using DataFrames; CSV.read(input, DataFrame; kw...)
│   caller = ip:0x0
└ @ Core :-1
┌ Warning: Assignment to `WZ` in soft scope is ambiguous because a global variable by the same name exists: `WZ` will be treated as a new local. Disambiguate by using `local WZ` to suppress this warning or `global WZ` to assign to the existing global variable.
└ @ ~/stephenz/locaTE-paper/tools/SCODE/SCODE.jl:69
┌ Warning: `writeheader=false` is deprecated in favor of `header=false`
│   caller = ip:0x0
└ @ Core :-1
a
mkdir: cannot create directory ‘SCODE_D_8’: File exists
┌ Warning: `CSV.read(input; kw...)` is deprecated in favor of `using DataFrames; CSV.read(input, DataFrame; kw...)
│   caller = ip:0x0
└ @ Core :-1
┌ Warning: Assignment to `WZ` in soft scope is ambiguous because a global variable by the same name exists: `WZ` will be treated as a new local. Disambiguate by using `local WZ` to suppress this warning or `global WZ` to assign to the existing global variable.
└ @ ~/stephenz/locaTE-paper/tools/SCODE/SCODE.jl:69
┌ Warning: `writeheader=false` is deprecated in favor of `header=false`
│   caller = ip:0x0
└ @ Core :-1
a
/data/gpfs/projects/punim0638/stephenz/miniconda3/lib/python3.10/site-packages/Scribe-0.0.0-py3.10.egg/Scribe/causal_network.py:18: UserWarning:  WARNING: No expression data argument given. if you intend to load the data from a file, call the method 'read_expression_file'.
  if expression==None: warnings.warn(" WARNING: No expression data argument given. if you intend to load the data from a file, call the method 'read_expression_file'.")
/data/gpfs/projects/punim0638/stephenz/miniconda3/lib/python3.10/site-packages/Scribe-0.0.0-py3.10.egg/Scribe/causal_network.py:298: FutureWarning: Could not cast to int64, falling back to object. This behavior is deprecated. In a future version, when a dtype is passed to 'DataFrame', either all columns will be cast to that dtype, or a TypeError will be raised.
  max_rdi_delay = pd.DataFrame({node_id: [np.nan for i in self.node_ids] for node_id in self.node_ids}, index=self.node_ids,
Create causal_model successfully
  Activating project at `/data/gpfs/projects/punim0638/stephenz/locaTE-paper/data/simulated/Synthetic_1000/dyn-CY/dyn-CY-1000-5/pidc_env`
    Updating registry at `~/.julia/registries/General.toml`
   Resolving package versions...
  No Changes to `/data/gpfs/projects/punim0638/stephenz/locaTE-paper/data/simulated/Synthetic_1000/dyn-CY/dyn-CY-1000-5/pidc_env/Project.toml`
  No Changes to `/data/gpfs/projects/punim0638/stephenz/locaTE-paper/data/simulated/Synthetic_1000/dyn-CY/dyn-CY-1000-5/pidc_env/Manifest.toml`
Getting nodes...
Inferring network...
